include/seqan3/argument_parser/validators.hpp

```diff
@@ -217,26 +243,45 @@ class value_list_validator
 private:
 
     //!\brief Minimum of the range to test.
-    std::vector<value_type> values;
+    std::vector<value_type> values{};
 };
 
 /*!\brief Type deduction guides
  * \relates seqan3::value_list_validator
  * \{
  */
-//!\brief Deduction guide for `std::vector` over an arithmetic type.
-template <arithmetic option_value_type>
-value_list_validator(std::vector<option_value_type>) -> value_list_validator<double>;
-
-//!\brief Deduction guide for `std::initializer_list` over an arithmetic type.
-template <arithmetic option_value_type>
-value_list_validator(std::initializer_list<option_value_type>) -> value_list_validator<double>;
+//!\brief Deduction guide for a parameter pack.
+template <typename first_type, typename ...option_types>
+    requires !std::same_as<first_type, const char *> && !arithmetic<first_type>
+value_list_validator(first_type, option_types...) -> value_list_validator<first_type>;
+
+//!\brief Deduction guide for a parameter pack over an arithmetic type.
+template <typename ...option_types>
+    requires (arithmetic<option_types> && ...)
```

Comment:
```suggestion
template <arithmetic ...option_types>
```
Or does this trigger a visibility bug 🤔

Reply:
@marehr I am not sure if this triggers the bug?

Reply:
This should be fine as it is.